apps/wing/src/commands/compile.test.ts

```diff
@@ -1,14 +1,23 @@
-import { Target, compile } from "./compile";
-import { readdir, stat, writeFile } from "fs/promises";
-import { describe, test, expect } from "vitest";
-import { join, resolve } from "path";
+import {Target, compile} from "./compile";
```

Comment:
Looks like you have a formatting issue in your local env. All these spaces before/after import names should remain